<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Configuration</class>
 <widget class="QWidget" name="Configuration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration de la partie</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="LabelConfiguration">
       <property name="font">
        <font>
         <pointsize>19</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Configuration</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="LabePremierConnecte">
       <property name="text">
        <string>Vous etes le premier connect√©.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="LabelStatut">
       <property name="text">
        <string>Veuillez configurer la partie.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="LabelNomPartie">
     <property name="text">
      <string>Nom de la partie :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="LineEditNomPartie">
     <property name="inputMask">
      <string/>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="LabelNbJoueurs">
       <property name="text">
        <string>Nombre de joueurs :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="SpinBoxNbJoueurs">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>16</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="LabelPseudoAdmin">
     <property name="text">
      <string>Pseudo :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="LineEditPseudoAdmin">
     <property name="maxLength">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="BoutonValider">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Valider</string>
       </property>
       <property name="icon">
        <iconset resource="ressouces.qrc">
         <normaloff>:/res/res/64/Dialog-Apply-64.png</normaloff>:/res/res/64/Dialog-Apply-64.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Loader">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ressouces.qrc"/>
 </resources>
 <connections/>
</ui>
